econml/dml.py

```diff
@@ -371,6 +404,49 @@ def __init__(self,
                          n_splits=n_splits,
                          random_state=random_state)
 
+    def fit(self, Y, T, X=None, W=None, sample_weight=None, inference=None):
+        """
+        Estimate the counterfactual model from data, i.e. estimates functions τ(·,·,·), ∂τ(·,·).
+
+        Parameters
+        ----------
+        Y: (n × d_y) matrix or vector of length n
+            Outcomes for each sample
+        T: (n × dₜ) matrix or vector of length n
+            Treatments for each sample
+        X: optional (n × dₓ) matrix
+            Features for each sample
+        W: optional (n × d_w) matrix
+            Controls for each sample
+        sample_weight: optional (n,) vector
+            Weights for each row
+        inference: string, `Inference` instance, or None
+            Method for performing inference.  This estimator supports 'bootstrap'
+            (or an instance of :class:`.BootstrapInference`) and 'debiasedlasso'
+            (or an instance of :class:`.LinearCateInference`)
+
+        Returns
+        -------
+        self
+        """
+        # TODO: support sample_var
```

Comment:
Seems important for the case where we have sample weights, otherwise we have an inconsistency between the linear dml and the sparse linear dml. Maybe we should raise a warning if sample_weight is not None, that inference does not take into account "within sample" variance and the feature will be updated in a future release.

Reply:
Done!